mono/metadata/class-internals.h

```diff
@@ -1592,6 +1592,59 @@ m_field_get_offset (MonoClassField *field)
 	return field->offset;
 }
 
+/*
+ * Memory allocation for classes/methods
+ *
+ *   These should be used to allocate memory whose lifetime is equal to
+ * the lifetime of the domain+class/method pair.
+ */
+
+static inline MonoMemoryManager*
```

Comment:
```suggestion
static inline MonoMemoryManager *
```

Reply:
We usually don't put spaces between those, only in variables, not in return values.